nanofi/ecu/coap_server.c

```diff
@@ -0,0 +1,138 @@
+#include <nanofi/coap_server.h>
```

Comment:
Multiple issues here:

- License
- What is the goal of this file? Are we trying to emulate a C2 server? Some comments would be nice
- A server including the header of the agent is a bad smell

Reply:
This file was initially not supposed to be pushed to the repo. I wrote it to test the c2 operations on ecus without having to have any other external tool but realized that it could be handy for anyone to test their ecus if they write one. Same goes for c2_client.c.

I will mention the purpose of each in the respective file and make the code much cleaner.